nhais-adaptor/adaptor/incoming/operation_definition_adaptor.py

```diff
@@ -0,0 +1,58 @@
+from edifact.incoming.models.interchange import Interchange
+import adaptor.fhir_helpers.fhir_creators as creators
+from datetime import datetime
+
+
+def format_date_time(edifact_date_time):
+    """
+    format the edifact date time to a fhir format
+    :param edifact_date_time: The incoming date time stamp from the interchange header
+    :return: The formatted date time stamp
+    """
+    current_format = "%y%m%d:%H%M"
+    desired_format = "%Y-%m-%d %H:%M"
+    formatted_date = datetime.strptime(edifact_date_time, current_format).strftime(desired_format)
+    return formatted_date
+

```

Comment:
Could this go with the other date formatters in the helper class?

Reply:
👍 yes it does. Will move

Reply:
Discussed with @Philip-Woods and agreed I will do this as part of the next ticket RED-35.